assets/vue/components/PageBlocked.vue

```diff
@@ -11,6 +11,13 @@
           id="inlineSearch"
           placeholder="Search for Incident/Package"
         />
+        <input
+          v-model="groupNames"
+          type="text"
+          class="form-control"
+          id="inlineSearch"
+          placeholder="Search for Group Names"
```

Comment:
```suggestion
          placeholder="Search for group names"
```
Although you might not want to follow that suggestion for the sake of consistency with existing placeholders. I'd prefer to fix the casing in all of them.